linux_os/guide/system/selinux/selinux_policytype/tests/selinuxtype_minimum.fail.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# profiles = xccdf_org.ssgproject.content_profile_C2S, xccdf_org.ssgproject.content_profile_ospp
+
+SELINUX_FILE='/etc/selinux/config'
+
+if grep -s '^[[:space:]]*SELINUXTYPE' $SELINUX_FILE; then
+	sed -i 's/^\([[:space:]]*SELINUXTYPE[[:space:]]*=[[:space:]]*\).*/\minimum/' $SELINUX_FILE
+else
+	echo 'SELINUXTYPE=minimum' >> $SELINUX_FILE
+fi
```

Comment:
The test is fine.

But as a side note, I think in general the test scenarios can be more direct and destructive.
Like for example, just remove the whole file and add the line subject of the test.
Thoughts on this?

Reply:
Agree, btw the sed command is wrong, should be `../\1minimum/`.

Reply:
`../\1minimum/` will fix.

Agree with the test scenario approach. But I followed the pattern of other tests.

But maybe, sometimes this destructive approach can be dangerous, for example... if we delete a file that contain other configurations that are required to make the system work then we may end with an inaccessible system.

Funny thing, I was testing if the selinux configuration was case insensitive and for one case it was invalid and I couldn't access the system and it took a while to figure it out ;( .

I would rather keep the test as it is right now. But for future ones I can consider this way of thinking.

Reply:
Kids, don't this at home:

`sed -i 's/^\([[:space:]]*SELINUXTYPE[[:space:]]*=[[:space:]]*\).*/\1taRgEteD/' /etc/selinux/config` 
```
         Starting Switch Root...
[    2.821988] systemd-journald[207]: Received SIGTERM from PID 1 (systemd).
[    2.841567] systemd: 13 output lines suppressed due to ratelimiting
[    2.856533] systemd[1]: Failed to load SELinux policy.
[!!!!!!] Failed to load SELinux policy, freezing.
[    2.861912] systemd[1]: Freezing execution.
[    3.182289] pciehp 0000:00:01.6:pcie004: Failed to check link status
```

Reply:
When I fixed the `minimum` sed, the test scenario started to fail on my RHEL8.1 VM after a restart. I'll remove it.